### PR TITLE
Fix hyperlinks, typos, restructure some proxy info

### PR DESCRIPTION
**Pull request by smaher-edb**
## What Changed?

- fixed links
- typos
- restructure proxy page

### CODE REVIEW

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx [hunk 1]
```diff
@@ -25,25 +24,14 @@ If PGD cluster is being deployed through TPA then it installs and configures PGD
...
-```sh
-# for Debian
-sudo apt-get install edb-pgd5-proxy
-```
-
-
-## Configuration
-
-Proxy connects to BDR database for its internal operations, like getting proxy configs, getting write leader details, etc. Therefore, it needs a list of endpoints/dsn to connect to BDR nodes. Proxy expects these configurations in a local config file `pgd-proxy-config.yml`. Following is a functional example of `pgd-proxy-config.yml` file:
+Proxy connects to BDR database for its internal operations, like getting proxy options, getting write leader details, etc. Therefore, it needs a list of endpoints/dsn to connect to BDR nodes. Proxy expects these configurations in a local config file `pgd-proxy-config.yml`. Following is a functional example of `pgd-proxy-config.yml` file:
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/installing_proxy.mdx:29]:
```suggestion
Proxy connects to the BDR database for its internal operations, like getting proxy options, getting write leader details, etc. Therefore, it needs a list of endpoints/dsn to connect to BDR nodes. Proxy expects these configurations in a local config file `pgd-proxy-config.yml`. Following is a functional example of `pgd-proxy-config.yml` file:
```

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx [hunk 2]
```diff
@@ -52,35 +40,35 @@ cluster:
     name: "proxy-a1"
 ```
 
-The `cluster.endpoints` and `cluster.proxy.name` are mandatory fields in the config file. Proxy always tries to connect to the first endpoint in the list, if it fails it tries the next endpoint and so on. The log level for PGD Proxy service can be set using the top level config parameter `log-level` as shown in the sample config. The valid values for `log-level` are `debug`, `info`, `warn` and  `error`.
+By default, in cluster created through TPA, `pgd-proxy-config.yml` is located in the `/etc/edb/pgd-proxy` directory. The PGD Proxy searches for `pgd-proxy-config.yml` in the following locations. Precedence order is high to low.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/installing_proxy.mdx:43]:
```suggestion
By default, in the cluster created through TPA, `pgd-proxy-config.yml` is located in the `/etc/edb/pgd-proxy` directory. The PGD Proxy searches for `pgd-proxy-config.yml` in the following locations. Precedence order is high to low.
```

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx [hunk 3]
```diff
@@ -52,35 +40,35 @@ cluster:
...
+  1. `/etc/edb/pgd-proxy` (default)
+  2. `$HOME/.edb/pgd-proxy`
 
-   1. "-f config-file" (e.g. 'pgd-proxy -f /opt/my-config.yml')
-   2. "/etc/edb/pgd-proxy" (default)
-   3. "$HOME/.edb/pgd-proxy"
+If you rename the file or move it to another location, specify the new name and location using the optional `-f` or `--config-file` flag when starting a service. See the [sample service file](#pgd-proxy-service).
 
-The `pgd-proxy-config.yml`, is located in the `/etc/edb/pgd-proxy` directory, by default. If you rename the file or move it to another location, specify the new name and location using the `-f` or `--config-file` flag in the `pgd-proxy.service` file.
+The log level for PGD Proxy service can be set using the top level config parameter `log-level` as shown in the sample config. The valid values for `log-level` are `debug`, `info`, `warn` and  `error`.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/installing_proxy.mdx:50]:
```suggestion
The log level for the PGD Proxy service can be set using the top level config parameter `log-level` as shown in the sample config. The valid values for `log-level` are `debug`, `info`, `warn` and  `error`.
```

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx [hunk 4]
```diff
@@ -52,35 +40,35 @@ cluster:
...
 
-### Proxy user
+The `cluster.endpoints` and `cluster.proxy.name` are mandatory fields in the config file. Proxy always tries to connect to the first endpoint in the list, if it fails it tries the next endpoint and so on.
 
-If the PGD Cluster is created by TPA, a postgres user `pgdproxy` for PGD Proxy and `route_dsn` at node level is set. If you wish to setup a different user then you need to create the user and set in the `endpoints` in config file manually.
+Proxy uses endpoints given in the local config file only at the time of proxy startup. After that, proxy retrieves the list of actual endpoints (route_dsn) from bdr catalog. Therefore, the node option - `route_dsn`, must be set for each bdr node. See [route_dsn](routing#configuration) for more information.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/installing_proxy.mdx:54]:
```suggestion
Proxy uses endpoints given in the local config file only at the time of proxy startup. After that, proxy retrieves the list of actual endpoints (route_dsn) from the bdr catalog. Therefore, the node option - `route_dsn`, must be set for each bdr node. See [route_dsn](routing#configuration) for more information.
```

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx [hunk 5]
```diff
@@ -52,35 +40,35 @@ cluster:
...
 
+#### Proxy user
 
-## PGD Proxy service
+Database user specified in the endpoint doesn’t need to be a superuser. Typically, in the TPAexec environment, `pgdproxy` is an OS user as well as database user with `bdr_superuser` role.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/installing_proxy.mdx:58]:
```suggestion
The database user specified in the endpoint doesn’t need to be a superuser. Typically, in the TPA environment, `pgdproxy` is an OS user as well as database user with the `bdr_superuser` role.
```

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx [hunk 6]
```diff
@@ -93,8 +81,18 @@ WantedBy=multi-user.target
...
+
+```sh
 systemctl status pgd-proxy
 systemctl stop pgd-proxy
 systemctl restart pgd-proxy
 ```
+
+### Installing manually
+
+You can manually install the PGD Proxy on any Linux machine using `.deb` and `.rpm` packages available from the PGD repository. The package name is `edb-pgd5-proxy`. For example:
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/installing_proxy.mdx:93]:
```suggestion
You can manually install PGD Proxy on any Linux machine using `.deb` and `.rpm` packages available from the PGD repository. The package name is `edb-pgd5-proxy`. For example:
```

File: product_docs/docs/pgd/5/routing/proxy.mdx [hunk 7]
```diff
@@ -26,33 +25,26 @@ PGD Proxy responds to write leader change events that can be categorized into tw
...
-
-Please note switchover may or may not happen to the user provided target leader. It depends on but not limited to following factors -
-- switchover command arguments like switchover method, timeout,
-- different group routing option values like route_writer_max_lag, and
-- data lags across nodes when switchover command was executed
-
+User controlled, manual transfer of write leadership from current write leader to new target leader is termed as `Switchover`.  Switchover is triggered through [PGD CLI switchover](cli/command_ref/pgd_switchover) command. The command is submitted to BDR which attempts to elect the given target node as new write leader. Similar to failover, proxy will close existing client connections and redirect new client connections to newly elected write leader. This is useful at the time of server maintenance, e.g. if the current write leader node needs to be stopped for maintenance like server update, OS patch update etc.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/proxy.mdx:28]:
```suggestion
User controlled, manual transfer of write leadership from the current write leader to a new target leader is termed as `Switchover`.  Switchover is triggered through the [PGD CLI switchover](cli/command_ref/pgd_switchover) command. The command is submitted to BDR which attempts to elect the given target node as new write leader. Similar to failover, proxy will close existing client connections and redirect new client connections to the newly elected write leader. This is useful at the time of server maintenance, e.g. if the current write leader node needs to be stopped for maintenance like a server update, OS patch update etc.
```

File: product_docs/docs/pgd/5/routing/proxy.mdx [hunk 8]
```diff
@@ -26,33 +25,26 @@ PGD Proxy responds to write leader change events that can be categorized into tw
...
 
 ## Managing PGD Proxy
 
-PGD CLI provides few commands viz., `create-proxy`, `show-proxy`, `set-proxy-options` and `delete-proxy` to manage Proxies in PGD cluster. See [PGD CLI](/product_docs/docs/pgd/5/cli/command_ref) for more information.
-
-See [Connection management](/product_docs/docs/pgd/5/routing) for more information on BDR side of configuration and management of PGD Proxy.
+PGD CLI provides few commands to manage proxies in PGD cluster such as `create-proxy`, `delete-proxy`, `set-proxy-options` and `show-proxies`. See [PGD CLI](cli/command_ref) for more information.
```
Comment (drothery-edb) [inline product_docs/docs/pgd/5/routing/proxy.mdx:32]:
```suggestion
PGD CLI provides a few commands to manage proxies in PGD cluster such as `create-proxy`, `delete-proxy`, `set-proxy-options` and `show-proxies`. See [PGD CLI](cli/command_ref) for more information.
```

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx (hunk 1 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/installing_proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx (hunk 2 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/installing_proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx (hunk 4 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/installing_proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx (hunk 5 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/installing_proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/proxy.mdx (hunk 7 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/proxy.mdx (hunk 8 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx (hunk 6 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/installing_proxy.mdx]: fixed

File: product_docs/docs/pgd/5/routing/installing_proxy.mdx (hunk 3 above)
Comment (smaher-edb) [reply product_docs/docs/pgd/5/routing/installing_proxy.mdx]: fixed

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**